driver/src/main/java/oracle/nosql/driver/NoSQLHandleConfig.java

```diff
@@ -142,6 +143,13 @@ public class NoSQLHandleConfig implements Cloneable {
      */
     private int maxChunkSize = 0;
 
+    /**
+     * Default http protocols
+     *
+     * Default: prefer H2 but fallback to Http1.1
+     */
+    private List<String> httpProtocols = new ArrayList<>(Arrays.asList(ApplicationProtocolNames.HTTP_2, ApplicationProtocolNames.HTTP_1_1));
```

Comment:
While we may need to use ApplicationProtocolNames when talking to Netty there should be no netty class or constant that is part of our own public API. We need to provide our own constants to represent http/2 and 1.1 and map them internally (probably not in this class) to netty when needed